ci-bootstrap/terraform/role.tf.tmpl

```diff
@@ -0,0 +1,87 @@
+# Run via ci-bootstrap/bin/deploy.sh
+
+provider "aws" {
+    region = "eu-west-2"
+}
+
+variable "environment" {}
```

Comment:
Can now add terraform backend s3 state storage as the bucket will have already been bootstrapped before this is run.

For example see https://github.com/ONSdigital/statement-of-record-terraform/pull/22